featuretools/feature_base/feature_base.py

```diff
@@ -436,7 +444,10 @@ def variable(self):
 
     @property
     def number_output_features(self):
-        return self.base_features[0].primitive.number_output_features
+        if isinstance(self.base_features[0], MultiOutputFeature):
```

Comment:
We might be able to just return `self.base_features[0].number_output_features` in all cases here